_posts/2019-02-08-resize-images-on-the-fly.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "How to resize your images on-the-fly with OpenFaaS"
+description: In this post Tarun from IconScout walks us through how his company is leveraging OpenFaaS with Node.js and OpenFaaS to boost image resizing and conversion speeds of up to 4-5x.
+date: 2019-02-04
+image: https://images.pexels.com/photos/440731/pexels-photo-440731.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260 # temporary
+categories:
+  - end-user
+  - node
+  - tutorial
+author_staff_member: alex
+dark_background: false
+---
+
+I’d like to introduce you to Tarun who is our guest-writer for today’s end-user blog post.
+
+> Author bio: [Tarun Mangukiya](https://twitter.com/tarunmangukiya?lang=en-gb) is co-founder and Product Manager at [IconScout](https://iconscout.com). He’s working on Frontend, Docker, and Serverless and many more. They are working to migrate their codebase to serverless over the time. Currently, their all the processing jobs are powered by OpenFaaS.
```

Comment:
They are working -> IconScout are working

Reply:
`He’s working on Frontend, Docker, and Serverless and many more.`

needs a slight reword

Reply:
Frontend and Serverless.